### PR TITLE
URL and Title should be in the index

### PR DESCRIPTION
**Pull request by ampaze**
This makes it possible to give the title a boost via the search query. 

As discussed here https://github.com/sulu/sulu/issues/5290

### CODE REVIEW

**Comment by alexander-schranz** [comment]
@ampaze is this PR needed if we implement something like https://github.com/sulu/sulu/pull/5304/files to mark which fields should be indexed?

**Comment by ampaze** [comment]
It would of course be better to be able to mark the fields like in https://github.com/sulu/sulu/pull/5304/files, but this requires more work.

Although I don't think there is any negative consequences in just alway indexing it.

File: Search/Adapter/ZendLuceneAdapter.php [hunk 1]
```diff
@@ -162,8 +162,8 @@ public function index(Document $document, $indexName)
         $luceneDocument->addField(
             Lucene\Document\Field::unStored(self::AGGREGATED_INDEXED_CONTENT, implode(' ', $aggregateValues))
         );
-        $luceneDocument->addField(Lucene\Document\Field::unIndexed(self::URL_FIELDNAME, $document->getUrl()));
-        $luceneDocument->addField(Lucene\Document\Field::unIndexed(self::TITLE_FIELDNAME, $document->getTitle()));
+        $luceneDocument->addField(Lucene\Document\Field::text(self::URL_FIELDNAME, $document->getUrl()));
+        $luceneDocument->addField(Lucene\Document\Field::text(self::TITLE_FIELDNAME, $document->getTitle()));
```
Comment (danrot) [inline Search/Adapter/ZendLuceneAdapter.php:166]: @chirimoya I think you have the most experience with running Zend Lucene in a production environment from us. Do you think that has a big performance impact?

**Comment by ampaze** [comment]
I am closing this in favor of https://github.com/sulu/sulu/pull/5304. 

Then use title instead of __title, which is a better solution anyway.

**Pull request closed without merging**